app/build.gradle

```diff
@@ -242,7 +242,7 @@ dependencies {
     // https://mvnrepository.com/artifact/junit/junit
     testImplementation 'junit:junit:4.13.2'
     // https://github.com/robolectric/robolectric
-    testImplementation "org.robolectric:robolectric:4.5.1"
+    testImplementation "org.robolectric:robolectric:4.6.1"
```

Comment:
Don't do it, tests will fail. 
It's bugged version, all devs are using 4.5.1

Reply:
ohh I see 
ok will revert it 👌🏻👍🏻

Reply:
why do we need robolectric:robolectric at first place?

Reply:
I meant do we use roboelectic in our current unit tests?

Reply:
@1sakov ^^ ?

Reply:
@1sakov fixed! ^^

Reply:
we do